crypto/src/mls/conversation/decrypt.rs

```diff
@@ -123,13 +124,17 @@ impl MlsConversation {
         callbacks: Option<&dyn CoreCryptoCallbacks>,
         restore_pending: bool,
     ) -> CryptoResult<MlsConversationDecryptMessage> {
-        // handles the crooked case where we receive our own commits.
-        // Since this would result in an error in openmls, we handle it here
```

Comment:
Is this (old) comment still relevant? Which error do we get from openmls if we let own commits through?

Reply:
The error that we expect and handle below

Reply:
Ah, I assume this is exactly the error you're matching on below, right?